health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,325 @@
+                  selectedBaseMapName={selectedBaseMapName}
+                  basemapRef={basemapRef}
+                  t={t}
+                />
+              </div>
+            </div>
+            <div className="bottom-left-map-subcomponents">
+              <div className="north-arrow">{DigitSvgs.NorthArrow && <DigitSvgs.NorthArrow width={"40px"} height={"40px"} fill={"#FFFFFF"} />}</div>
+              <CustomScaleControl map={map} t={t} />
+            </div>
+            <div className="bottom-right-map-subcomponents">
+              <ViewElement label={type} type={type}></ViewElement>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)